apps/signing/views.py

```diff
@@ -85,6 +86,7 @@ def put(self, request, addon, version_string):
                         status=status_code)
 
     @with_addon()
+    @use_master
```

Comment:
ooh, yes it should. But I think decorators are applied inside out. Let me think about this while it melts my brain.